matrix_openCL/Project1/vector_adds_kernel.cl

```diff
@@ -0,0 +1,8 @@
+__kernel void vector_add(__global const int *A, __global const int *B, __global int *C) {
+ 
+    // Get the index of the current element to be processed
+    int i = get_global_id(0);
```

Comment:
I see that i gets all the values from 0 to LIST_SIZE, but i don't understand why.

Reply:
this gets the ID of the work item. in our example here we create LIST_SIZE work items, and this ID is used as the index in the vectors.
the zero parameter here is the dimension for which to get the ID. there are supported 3 dimensions fro . what I've read. here we are working with vectors, so just one dimension.